components/html-block/test/html-block.vdiff.js

```diff
@@ -149,8 +149,16 @@ describe('d2l-html-block', () => {
 
 	it('update-content', async() => {
 		const elem = await fixture(html`<d2l-html-block html="before update"></d2l-html-block>`, { viewport });
+
+		let resolve, runCount = 0;
+		const elemUpdated = new Promise(r => resolve = r);
+		elem.updated = async function(changedProperties) {
+			await Object.getPrototypeOf(elem).updated.call(elem, changedProperties);
+			if (++runCount === 2) resolve();
```

Comment:
Just realizing this could probably just check for `changedProperties.has('embeds')` now instad of counting.

Reply:
Ah yeah, good point.